DOCS-1363: Clarify secondary org owner location sharing (#2339)
viamrobotics/docs: docs/fleet/locations.md

A location always belongs to the organization it was created in.
Members of the organization have access to all locations in the organization by default.

You can share a location beyond its organization by sharing a location with an additional organization:

#### Share a location with an additional organization

Share your location with another organization you belong to by selecting the organization from the **Add Organization** dropdown menu and clicking **Share**.

> To share your location with an organization you are not a member of, enter the organization ID (a string like `1ab2c3d1-1234-123a-abcd-abcdef123456`) and click **Share**.

Reviewer: Will it show you how to share it with an org you are a member of?

Author: See sentence above this--do you think it needs changing?

Members of the other org can find the org ID on their org settings page.

![The Add Organization dropdown in the Viam app displays all organizations the user is a member of.](/fleet/app-usage/add-org-drop-down.png)

The location's page lists the newly added organization, along with the organization identified as the _primary owner_:

![After adding another org, the Viam app lists it under the orgs that share this location list.](/fleet/app-usage/after-add-org.png)

The list of organizations includes the names of organizations to which you belong, and the org IDs of organizations to which you do not belong.

{{% alert title="Info" color="info" %}}

> Only members of the _primary owner_ organization can share the location.

Reviewer: Only owners of the primary org, owners location, and owners in the secondary org can share the location

Author: So if Natalia's Org owns Location 123, and you share Location 123 with Jessamy's Org, then I can share Location 123 with Brunhilda's Org? And then can Brunhilda share Location 123 with Damian's Org or does the chain stop between secondary and tertiary org owners?

> If the location has been shared with additional organizations, members of those secondary organizations cannot share the location.

Reviewer: This is not correct, they can as long as they are org owners in their org

Author: So if Natalia's Org owns Location 123, and you share Location 123 with Jessamy's Org, then I can share Location 123 with Brunhilda's Org? And then can Brunhilda share Location 123 with Damian's Org or does the chain stop between secondary and tertiary org owners?

Author: ```suggestion
If the location has been shared with additional organizations, owners of those secondary organizations can also share the location.
```
